src/core/schema/contentobject.model.schema

```diff
@@ -0,0 +1,118 @@
+{
+  "type":"object",
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "id": "http://jsonschema.net",
+  "$ref": "http://localhost/system/content.schema",
+  "properties": {
+    "title": {
+      "type":"string",
+      "required":true,
+      "default": "New Menu/Page Title",
+      "inputType":"Text",
+      "validators": ["required"],
+      "translatable": true
+    },
+    "displayTitle": {
+      "type":"string",
+      "required":false,
+      "default": "New Menu/Page Title",
+      "inputType":"DisplayTitle",
+      "validators": [],
+      "help": "When viewing a menu/page - this is the title that will be displayed on the menu/page",
+      "translatable": true
+    },
+    "body":{
```

Comment:
should be added in this PR https://github.com/adaptlearning/adapt_framework/pull/1271